### ISSUE TITLE
As a user I want to mark videos as viewed and courses as completed

### ISSUE THREAD

**Issue by federicosan** [scrimba/community#507]
As a user I want to mark videos as viewed and courses as completed

**Comment by drumrock** [comment]
Agree. Maybe viewed lessons could be automatically grayed in playlist.

**Comment by azivkovi** [comment]
Some parts of the course I would rather skip because of prior knowledge or other reason. Would be great if I could mark them complete on the course outline page, where I see 100% for the lessons completed.
